react/EmptyStateMessage.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, Fragment } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { useCssHandles } from 'vtex.css-handles'
+
+import EmptyStateIcon from './components/EmptyStateIcon'
+
+const CSS_HANDLES = ['emptyStateIconWrapper', 'emptyStateMessage'] as const
+
+const EmptyStateMessage: FC = () => {
+  const handles = useCssHandles(CSS_HANDLES)
+
+  return (
+    <Fragment>
+      <div className={`${handles.emptyStateIconWrapper} tc pv8`}>
+        <EmptyStateIcon />
+      </div>
+      <p className={`${handles.emptyStateMessage} c-muted-1 pv9 tc`}>
+        <FormattedMessage id="store/minicart.empty-state" />
```

Comment:
I would suggest to add this as default and accept children to render something like a text and a link, but let's wait for someone ask for this

Reply:
Good idea! But I guess we're better off not creating this new block.